src/clj/athens/self_hosted/web/presence.clj

```diff
@@ -87,15 +94,23 @@
                                                                                     current-username
                                                                                     new-username)]
 
+    (swap! all-presence (fn [all]
+                          (-> all
+                              (update-in [:presence :users] set/rename-keys {current-username new-username})
+                              (update-in [:presence :users new-username] assoc :username new-username))))
     (clients/add-client! channel new-username)
     (clients/broadcast! broadcast-rename-event)
     (common-events/build-event-accepted id max-tx)))
 
 
 (defn goodbye-handler
-  [channel _event]
-  (let [_username (clients/get-client-username channel)]
-    (prn _event)))
+  [channel]
+  (let [username (clients/get-client-username channel)
+        ;; TODO: max-tx shouldn't be 42
```

Comment:
Presence events are forced to communicate `max-tx`,
this needs cleaning.
Maybe not part of this PR, but still need to clean it somewhere.

Reply:
Fixed for now by actually determining the right one from datahike.